tests/system_tests_tcp_adaptor.py

```diff
@@ -112,8 +123,9 @@ class EchoClientRunner:
     def __init__(self, test_name, client_n, logger, client, server, size,
                  count,
                  print_client_logs=True,
-                 timeout=TIMEOUT,
-                 port_override=None):
+                 timeout=TIMEOUT * 2,
```

Comment:
Why does this now require twice the time?

Reply:
When the TCP over TLS tests are run, the echo client stays up for a duration on 60 seconds for the entire test run. Sometimes, in the CentOS stream case, the echo client would abruptly stop after 60 seconds but the router is slow and still in the process of sending large messages back to the echo client. I extended the timeout for this reason. This failure became more pronounced after I added a test called test_14_tcp_EA2_EC1_2000000. I will set the timeout back to TIMEOUT  and see what happens on the PR branch